src/main/java/com/rettichlp/UnicacityAddon/events/HotkeyEventHandler.java

```diff
@@ -92,6 +93,18 @@ private void handleHotkey() {
             p.sendChatMessage("/aduty");
         } else if (Keyboard.isKeyDown(KeyBindRegistry.aDutySilent.getKeyCode())) {
             p.sendChatMessage("/aduty -s");
+        } else if (Keyboard.isKeyDown(KeyBindRegistry.freinforcement.getKeyCode())) {
+            BlockPos position = p.getPosition();
+            int posX = position.getX();
+            int posY = position.getY();
+            int posZ = position.getZ();
+            p.sendChatMessage("/f Benötige Verstärkung! -> X: " + posX + " | Y: " + posY + " | Z: " + posZ);
```

Comment:
- call position with position.getX() instead of posX (also for Y and Z)
- remove posX, posY and posZ